### PR TITLE
Helpers

### PR DESCRIPTION
**Pull request by bjartek**
The Metadata standard is very generic in its nature and I have found several error cases that I think warrants helpers to resolve the core views. 

One liner solutions can panic your entire system if an NFT say they resolve Display but return just a String. And since the interface is to return AnyStruct? they will get no compiler erorr when doing so.

### CODE REVIEW

**Comment by bshahid331** [comment]
+1 For this, this is something the catalog could leverage as well to extra views more safely. Changes LGTM.

File: contracts/MetadataViews.cdc [hunk 1]
```diff
@@ -70,6 +70,16 @@ pub contract MetadataViews {
         }
     }
 
+    ///a helper to get Display in a typesafe way
```
Comment (joshuahannan) [inline contracts/MetadataViews.cdc:73]:
```suggestion
    /// A helper to get Display in a typesafe way
```

Can you do this to all the comments?

**Review by joshuahannan — COMMENTED** [review]
Looks good! Just had one comment. Also, can you write some scripts/tests for these and run everything with `make test`?

File: contracts/MetadataViews.cdc (hunk 1 above)
Comment (bjartek) [reply contracts/MetadataViews.cdc]: Sure.

**Comment by bjartek** [comment]
> Looks good! Just had one comment. Also, can you write some scripts/tests for these and run everything with `make test`?

Would it be possible to do this with overflow? Are the generated files used anyhwere else?

**Review by joshuahannan — APPROVED** [review]
Thanks for updating the scripts. I want to integrate overflow, but we still need something for this I think: https://github.com/bjartek/overflow/issues/9

I'll see what I can do

**Comment by joshuahannan** [comment]
It looks like there are some merge conflicts also

**Comment by bjartek** [comment]
> Thanks for updating the scripts. I want to integrate overflow, but we still need something for this I think: [bjartek/overflow#9](https://github.com/bjartek/overflow/issues/9)
> 
> I'll see what I can do

We kind need to do a pr to flowkit Deploy code then to be able to specify a base path or something I think.

**Pull request merged**